ddtrace/settings/config.py

```diff
@@ -334,6 +334,7 @@ def __init__(self):
         self._ci_visibility_intelligent_testrunner_enabled = asbool(
             os.getenv("DD_CIVISIBILITY_ITR_ENABLED", default=False)
         )
+        self._ci_visibility_unittest_enabled = asbool(os.getenv("DD_CIVISIBILITY_UNITTEST_ENABLED", default=False))
```

Comment:
We need to document this configuration here: https://github.com/DataDog/dd-trace-py/blob/1.x/docs/configuration.rst?plain=1#L19

Reply:
hey @mabdinur. I don't think we need to instrument this feature flag because it's purely to avoid shipping a half finished feature to customers. This PR is only the first piece of the unittest visibility support (the last one is https://github.com/DataDog/dd-trace-py/pull/6611, which will remove this flag). Doing all the work in a single PR would've made it way harder to review and that's why we split it.